fe/fe-core/src/main/java/com/starrocks/sql/ast/UserIdentity.java

```diff
@@ -62,6 +62,8 @@ public class UserIdentity implements ParseNode, Writable, GsonPostProcessable {
     private String host;
     @SerializedName("isDomain")
     private boolean isDomain;
+    @SerializedName("eph")
```

Comment:
Is it necessary to serialize ephemeral fields? I understand this is just a volatile object that will be discarded after being used once?

Reply:
unnecessary, removed